### PR TITLE
Internationalize pages

### PR DESCRIPTION
**Pull request by arku**
# 💅🏼 What issue does this fix?

#23 #251

Replaces all hard-coded English copy with `t('key')` to support both English and Spanish.

## 🍂 Before You Submit

* [ ] Did you write tests?
* [ ] Did you run `bundle exec rspec` from the root?
* [ ] Did you run `bundle exec rails rswag` from the root?
* [ ] Did you run `bundle exec rubocop` from the root?
* [x] Did you run `yarn lint` in `/client`?
* [x] Did you run `yarn test` in `/client`?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment Considerations

* [ ] Data Migrations
* [ ] Schema Migrations
* [ ] Dependencies

## 🧳 Steps to test

Visit all the pages and switch the language (you can update the language in `localStorage` — `localStorage.setItem('i18nextLng', 'es')` or use the language switcher on the login and signup pages).

### CODE REVIEW

File: client/src/Signup/Confirmation.js [hunk 1]
```diff
@@ -24,33 +25,33 @@ ListItem.propTypes = {
...
+          <Text>{t('restartSignup', { userEmail })}</Text>
         </ListItem>
         <ListItem>
-          <Text>Check your spam folder</Text>
+          <Text>{t('checkSpam')}</Text>
         </ListItem>
         <ListItem>
           <Text>
-            Add <Text underline={true}>{pieEmail}</Text> to your contacts.
+            {t('add')} <Text underline={true}>{pieEmail}</Text>{' '}
```
Comment (arku) [inline client/src/Signup/Confirmation.js:49]: I couldn't do it with a single translation text as there isn't a straightforward way to interpolate HTML. If there's a simple way, do let me know.

Comment (katelovescode) [reply client/src/Signup/Confirmation.js:49]: This is how I would have done it.

File: client/src/i18n/locales/es/index.json [hunk 2]
```diff
@@ -48,7 +48,19 @@
   "passwordConfirmationMatch": "¡Las dos contraseñas que ingresaste no coinciden!",
   "passwordConfirmationPlaceholder": "Confirma tu contraseña",
   "termsRequired": "Lee y acepta las Condiciones de Uso",
+  "signupThanks": "¡Gracias por registrarse!",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:51]: ¡Gracias por registrarte!

File: client/src/i18n/locales/es/index.json [hunk 3]
```diff
@@ -48,7 +48,19 @@
...
+  "emailVerificationText": "We’ve sent you an email to verify your account.",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:52]: Te hemos mandado un correo para verificar tu cuenta.

File: client/src/i18n/locales/es/index.json [hunk 4]
```diff
@@ -48,7 +48,19 @@
...
+  "emailNotReceived": "Le hemos enviado un correo electrónico para verificar tu cuenta.",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:53]: Te hemos mandado un correo para verificar tu cuenta.

File: client/src/i18n/locales/es/index.json [hunk 5]
```diff
@@ -48,7 +48,19 @@
...
+  "restartSignup": "¿Es {userEmail} tu correo electrónico correcto sin errores de escritura? Si no, puedes reiniciar el proceso de registro",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:54]: "restartSignup": "¿Es {userEmail} tu correo correcto sin erratas? Si no, puedes reiniciar el proceso de registro",

File: client/src/i18n/locales/es/index.json [hunk 6]
```diff
@@ -48,7 +48,19 @@
...
+  "checkSpam": "Revisa tu carpeta de spam.",
+  "add": "Añade",
+  "addToContacts": "a tus contactos.",
+  "resendConfirmationEmail": "Haga clic aquí para reenviar el correo electrónico.",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:58]: "resendConfirmationEmail": "Haga clic aquí para reenviar el correo.",

File: client/src/i18n/locales/es/index.json [hunk 7]
```diff
@@ -62,5 +74,27 @@
   "gettingStartedInstructions": "Sigue estas instrucciones para configurar tu pantalla principal de casos. Esto debería tardar unos 15 minutos y solo tendrás que hacerlo una vez. ¡Prepárate para aumentar tu porción del pastel!",
   "gettingStartedButton": "Comienza",
 
-  "genericErrorMessage": "Algo salió mal, inténtalo de nuevo más tarde."
+  "genericErrorMessage": "Algo salió mal, inténtalo de nuevo más tarde.",
+
+  "uploadCases": "Subir los casos",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:79]: Sube tus casos

File: client/src/i18n/locales/es/index.json [hunk 8]
```diff
@@ -62,5 +74,27 @@
...
+  "uploadCasesInvalidFileFormat": "Formato de archivo inválido. Formatos admitidos: CSV, XLS y XLSX",
+  "reviewImportedCases": "Revisar los casos importados",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:81]: Revisa los casos importados

File: client/src/i18n/locales/es/index.json [hunk 9]
```diff
@@ -62,5 +74,27 @@
...
+
+  "firstName": "Nombre",
+  "lastName": "Apellido",
+  "dateOfBirth": "Fecha de nacimiento",
+  "siteId": "Sitio",
+  "caseStatus": "Estado del caso",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:87]: Estado de caso

File: client/src/i18n/locales/es/index.json [hunk 10]
```diff
@@ -62,5 +74,27 @@
...
+  "caseNumber": "Número de caso",
+  "fullDaysPerMonth": "Días completos (por mes)",
+  "partDaysPerMonth": "Días parciales (por mes)",
+  "effectiveOn": "Efectivo en",
+  "expiresOn": "Expira en",
+  "copay": "Copago",
+  "copayFrequency": "Frecuencia de copago",
+
+  "notFound": "404: No se encuentra",
+  "goBackHome": "Ir a inicio",
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:97]: Ve al inicio

File: client/src/i18n/locales/es/index.json [hunk 11]
```diff
@@ -62,5 +74,27 @@
...
+
+  "dashboardTitle": "Este es el dashboard"
```
Comment (csprayregen) [inline client/src/i18n/locales/es/index.json:99]: Este es la página principal

**Pull request merged**